hackathon_site/dashboard/frontend/src/components/general/PopupModal/PopupModal.module.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../assets/abstracts/mixins";
+@import "../../../assets/abstracts/variables";
+
+.modalSize {
+    position: absolute;
+    //margin: auto;
+    margin-top: 20px;
+    top: 40%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 400px;
```

Comment:
for the width, you should consider how this looks on different screens. Some phones have a width < 400px so the popup would overflow in that case and make the website look weird. On the other hand, if you're on a large monitor, the modal is very small. 
Use media queries (@media css decorator - look at examples in our code) with MUI breakpoints (e.g. sm-down for mobile, md-up for laptop/desktop) to adjust the width of the popup accordingly.